Lesson8/Task2/Lion.java

```diff
@@ -0,0 +1,35 @@
+package Lesson8.Task2;
+
+public class Lion extends Animal implements AdditionalPossibilities {
+    private String name = "Лев";
+    private double weight;
```

Comment:
не добре зразу імя давати
```suggestion
    private String name;
    private double weight;
```

Reply:
Згідна, дякую за підказку, 
я випадково пропустила одне ім'я